web/templates/invite_user_modal.hbs

```diff
@@ -3,20 +3,25 @@
     <div class="alert" id="dev_env_msg"></div>
     {{/if}}
     <div class="input-group">
-        <label for="invitee_emails">{{t "Emails (one on each line or comma-separated)" }}</label>
-        <textarea rows="2" id="invitee_emails" name="invitee_emails" class="invitee_emails" placeholder="{{t 'One or more email addresses...' }}"></textarea>
-        {{#if is_admin}}
-        <div id="invite-method-choice">
-            {{t "or" }} <a role="button" tabindex="0" id="generate_multiuse_invite_button">{{t "Generate invite link" }}</a>
+        <label>{{t "How would you like to invite users?" }}</label>
+        <div class="multiuse_radio_section prop-element" id="invite-user" data-setting-widget-type="radio-group" data-setting-choice-type="string">
```

Comment:
I don't think `multiuse_radio_section` is a good class name here, since it contains both options. Probably something like `invite_type_radio_section` would be better.

Reply:
And we would actually not need these `data-setting-widget-type` and `data-setting-choice-type` attributes here since they are used for save-discard widget code. Adding them does no harm, but I don't think they are needed.